### PR TITLE
API/DragEvent: add Constructor + fix Chrome/Opera version_added

### PR DESCRIPTION
**Pull request by caugner**
Sources:
1. Chromium bug tracker:
   https://crbug.com/498504
2. Chromium repository:
   chromium/chromium@77fa733
3. Firefox repository:
   mozilla/gecko-dev@a6d6c66

### CODE REVIEW

File: api/DragEvent.json [hunk 1]
```diff
@@ -47,6 +47,55 @@
           "deprecated": false
         }
       },
+      "DragEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DragEvent/DragEvent",
+          "description": "<code>DragEvent</code> constructor",
```
Comment (connorshea) [inline api/DragEvent.json]: Shouldn't this be `<code>DragEvent()</code> constructor`?

File: api/DragEvent.json [hunk 2]
```diff
@@ -47,6 +47,55 @@
...
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "3.5"
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
```
Comment (connorshea) [inline api/DragEvent.json]: Is there a reason to believe the MDN data table is incorrect? It says IE support started with IE10. https://developer.mozilla.org/en-US/docs/Web/API/DragEvent/DragEvent#Browser_compatibility

Comment (caugner) [reply api/DragEvent.json]: Oops, good catch! Fixed this in 6cf0f6a.

File: api/DragEvent.json (hunk 1 above)
Comment (caugner) [reply api/DragEvent.json]: I wasn't aware, but you're right, I fixed it in aa4a4e0 and will submit a PR to fix other occurrences of this mistake.

Comment (caugner) [reply api/DragEvent.json]: See #2458.

**Pull request merged**